--- a/linbox/matrix/sparsematrix/sparse-ell-matrix.h
+++ b/linbox/matrix/sparsematrix/sparse-ell-matrix.h
@@ -1205,6 +1205,7 @@ namespace LinBox
 				, _row(iter._row)
 			{}
 
+#if 0
 			_IndexedIterator &operator = (const _IndexedIterator &iter)
 			{
 				_colid_beg = iter._colid_beg ;
@@ -1218,6 +1219,7 @@ namespace LinBox
 
 				return *this;
 			}
+#endif
 
 			bool operator == (const _IndexedIterator &i) const
 			{


--- a/linbox/matrix/sparsematrix/sparse-ellr-matrix.h
+++ b/linbox/matrix/sparsematrix/sparse-ellr-matrix.h
@@ -1099,6 +1099,7 @@ namespace LinBox
 
 			{}
 
+#if 0
 			_Iterator &operator = (const _Iterator &iter)
 			{
 				_data_it  = iter._data_it  ;
@@ -1111,6 +1112,7 @@ namespace LinBox
 
 				return *this;
 			}
+#endif
 
 			bool operator == (const _Iterator &i) const
 			{
@@ -1246,6 +1248,7 @@ namespace LinBox
 				, _row(iter._row)
 			{}
 
+#if 0
 			_IndexedIterator &operator = (const _IndexedIterator &iter)
 			{
 				_rowid_it  = iter._rowid_it ;
@@ -1260,6 +1263,7 @@ namespace LinBox
 
 				return *this;
 			}
+#endif
 
 			bool operator == (const _IndexedIterator &i) const
 			{


--- a/linbox/vector/blas-subvector.h
+++ b/linbox/vector/blas-subvector.h
@@ -117,8 +117,10 @@ namespace LinBox {
          * \param dim dimension
          * \param inc distance between two element
          */
+#if 0
         BlasSubvector (Self_t &V, size_t beg, size_t inc, size_t dim) :
             _ptr(V.data()+beg), _size(dim), _inc(inc), _field(&V.field()) {}
+#endif
 
         
         /** Constructor from an existing @ref BlasVector
